ui/packages/catalog-ui-search/src/main/webapp/component/location-old/location-old.js

```diff
@@ -581,6 +571,8 @@ module.exports = Backbone.AssociatedModel.extend({
   },
 
   isLatLonValid: function(lat, lon) {
+    lat = parseFloat(lat)
+    lon = parseFloat(lon)
     return (
       lat !== undefined &&
```

Comment:
checking for undefined and NaN can be removed, number comparison already take care of that